rpcs3/Gui/MainFrame.cpp

```diff
@@ -337,6 +344,50 @@ void MainFrame::BootElf(wxCommandEvent& WXUNUSED(event))
 	LOG_SUCCESS(LOADER, "(S)ELF: boot done.");
 }
 
+void MainFrame::DecryptSPRXLibraries(wxCommandEvent& WXUNUSED(event))
+{
+	wxFileDialog ctrl(this, L"Select SPRX files", wxEmptyString, wxEmptyString,
+		"SPRX files (*.sprx)|*sprx",

```

Comment:
I just tested the case where the input files have the *.prx extension.

Here the results:
```
libsre.sprx  ->  libsre.prx
libsre.prx   ->  libsreprx
```

So we need to check for the file extension and adjust the ```prx_path.erase``` part. And to improve this even more, output the decrypted modules in a subfolder from where the input files comes from.

Just two questions: Can unsinged RPX also be encrypted? Does prx (not sprx) files are actually exists on a real PS3 firmware? Atleast, I never saw rpx files on a PS3 firmware, just sprx.

I mean would it be a profit to implement the case with *.prx input files?